Fallout2/Fallout1in2/Mapper/source/scripts/headers/debug.h

```diff
@@ -6,4 +6,11 @@
 #define debug_mode  (global_var(GVAR_ENABLE_DEBUG_MESSAGES) == 1)
 #define debug(x)    if debug_mode then display_msg(x)
 
+// Macros for debugging messages
+#ifdef DEBUG
+#define DEBUGMSG(x)       debug_msg(x);
+#else
+#define DEBUGMSG(x)       //nothing
+#endif

```

Comment:
This has to go; nothing worse than N ways to display debug spam, each working differently,

Reply:
What?

Reply:
Wipe means, it is a #define of a #define of a #define for a gvar condition. :>

Reply:
I mean we have debug() which can be switched on and off via .ini, there's no point introducing another basing on defines.

Reply:
debug(x) prints text to ingame message log, though while debug_msg(x) prints to debug.log :p

Reply:
`DEBUGMSG ()` - this based on conditional compilation, macro is set via compiler option [-mDEBUG](https://github.com/FakelsHub/int-sslc/blob/b97c184d09e4bd6285b9c5da4701fedb399483f6/scr/compile.c#L71)

Reply:
![2019-11-24_02-33-38](https://user-images.githubusercontent.com/21695363/69486966-0532fc00-0e63-11ea-9e91-7543c4ed0580.png)

Reply:
I know that, i just see no point introducing such way to turn on _part of_ debug messages.
Change .ini for this spam, compile with -m for other spam, bleh :/